### DISCUSSION TITLE
slow burndown.daily report

### DISCUSSION THREAD

**Discussion by gour** [GothenburgBitFactory/taskwarrior#3307] [Community support]
Hello!

While anticipating 3.0 to arrive into my distribution, I wonder if it should improve time to complete `burndown` reports?

For instance, here is what I currently get wit h2.6.2:

```
Executed in  167.89 secs    fish           external
   usr time   56.11 secs  307.00 micros   56.11 secs
   sys time  111.29 secs  144.00 micros  111.29 secs
```

which is useless for some interactive shell scripting...

**Comment by djmitche** [comment]
I don't know why that's slow, but I suspect it loads the entire taskdb into memory and then does some analysis. That loading process may get a bit slower, and the analysis probably won't change. So, I'd be surprised if it got faster!

I'm impressed: with 1.5y of data my burndown takes a bit over 1s to execute. You must have a lot of tasks in there!

**Reply by djmitche** [reply]
I moved this to #3412 to track it as a performance issue.
